src/intro-to-c/arrays/arrays-intro.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Array Introduction
+author:
+    - Bed Pandey
+    - Carson Cox
+---
+
+Arrays can be used to represent a collection of items with the same datatype. For instance, let's say we wanted an array to represent the grades our class got for a test in our hypothetical class of size 5.
+
+```
++------+------+------+------+------+
+| 92.5 | 52.4 | 75.0 | 82.0 | 88.9 |
++------+------+------+------+------+
+```
+
+This is a common representation of an array, a one-dimensional grid with a specific size. Since our array should have 5 grades, we have 5 separate cells we can put our data in.
+
+But how do we do this in C?!
+
+## Creating Arrays
+To create an array, we need to specify the datatype, a variable name, and the size of the array:
+
+```c
+float grades[5];
+```
+Here, we just created an array of size 5 with a datatype of `float`, meaning each element in this array should be a `float`.
+
+We can also have different types:
+
+```c
+int pineapples[10];     // integer array of size 10
+double coordinates[4];  // double array of size 4
+char word[4];           // char array of size 4 - also a string, sneak peak!
+```
+
+However, in our previous example, we only created the _skeleton_ that can hold our grades, but we haven't actually put anything in it. Our array is currently empty. So, how do we put our grades into our array?
+
+## Populating our Array
+Currently, our array (essentially) looks like this:
+
+```
++------+------+------+------+------+
+|      |      |      |      |      |
++------+------+------+------+------+
+```
+
+It's empty and barren. Let's put some grades into it!
```

Comment:
Not sure if you want to mention it early, or maybe just hint at it, but technically there is *something mysterious* there already...

Reply:
I thought a tiny bit about that (which is why the line right before says "essentially", though I guess the array wouldn't "essentially" look like that
I don't know if I want to go too deep into garbage values yet but honestly thinking back I think I was introduced to how arrays hold garbage values at the same time I was introduced to arrays. So you're probably right